### ISSUE TITLE
Closing server directly error(Press Ctrl+C/Alt+F4 or click X button)

### ISSUE THREAD

**Issue by Jaciezyt** [LiteLDev/LiteLoader#638]
### Exceptions module

ScriptEngine

### Operating System

Windows Server 2019

### LiteLoader version

81dda7c94 Private Build

### BDS version

1.19.10

### What happened?

When closing server directly(not use stop command), it will display the error below
```
Assertion failed: new_time >= loop->time, file D:\a\NodeJsBuilds\NodeJsBuilds\deps\uv\src\win\core.c, line 324
```

### Steps to reproduce?

Close the server by pressing ctrl+c

### Relevant log output

```shell
Assertion failed: new_time >= loop->time, file D:\a\NodeJsBuilds\NodeJsBuilds\deps\uv\src\win\core.c, line 324
```

**Comment by yqs112358** [comment]
DO NOT CLOSE SERVER BY CTRL+C 
unless you think it is ok to see a bit data loss & world broken

**Comment by Jaciezyt** [comment]
Fixed in ba48d0c04
